### PR TITLE
A user should see invitation (note) to create a first list in case there is no list created yet

### PR DESCRIPTION
**Pull request by anatoliikryvishyn**

### CODE REVIEW

**Comment by anatoliikryvishyn** [comment]
@serhiir, please review and  test original version [here](https://tada-interlink.herokuapp.com/) and version with no lists [here](https://trouble-tada-interlink.herokuapp.com/). But as we have no possibility to delete lists, if you will want to test again after list will be created, please contact me to clean data.

File: app/assets/javascripts/lists.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ var TodoList = function () {
 
             displayList(options.html);
 
+
             showItemCreationFormForNewList(options.id);
```
Comment (serhiir) [inline app/assets/javascripts/lists.js]: @anatoliikryvishyn Empty line? :)

File: app/assets/stylesheets/list.scss [hunk 2]
```diff
@@ -139,3 +139,38 @@ label {
...
+.no-list-notification-arrow-icon {
+  display: inline-block;
+  text-indent: -9999px;
+  width: 70px;
+  height: 82px;
+  background-image: asset_url('arrow.svg');
+  background-size: 70px 82px;
+  margin-bottom: 0px;
+
+}
```
Comment (serhiir) [inline app/assets/stylesheets/list.scss]: @anatoliikryvishyn Please remove empty line here.

**Comment by serhiir** [comment]
@anatoliikryvishyn There changes related to bootstrap. Did you changed/upgraded bootstrap and what is the purpose?

**Comment by serhiir** [comment]
@anatoliikryvishyn Done.

**Comment by anatoliikryvishyn** [comment]
@serhiir , the purpose of [this](https://github.com/anatoliikryvishyn/TaDa/pull/9#issuecomment-166327049) was that to use bootstrap glyphicons - you need to have bootstrap glyphicons fonts included.
But as we decided to use image icons - there is no need to have bootstrap fonts now, so I will delete them.

**Comment by anatoliikryvishyn** [comment]
@serhiir , updated.

File: vendor/assets/stylesheets/bootstrap.css [hunk 3]
```diff
@@ -263,8 +263,8 @@ th {
 @font-face {
   font-family: 'Glyphicons Halflings';
 
-  src: url('../fonts/glyphicons-halflings-regular.eot');
-  src: url('../fonts/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'), url('../fonts/glyphicons-halflings-regular.woff2') format('woff2'), url('../fonts/glyphicons-halflings-regular.woff') format('woff'), url('../fonts/glyphicons-halflings-regular.ttf') format('truetype'), url('../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
+  src: url('bootstrap/glyphicons-halflings-regular.eot');
+  src: url('bootstrap/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'), url('bootstrap/glyphicons-halflings-regular.woff2') format('woff2'), url('bootstrap/glyphicons-halflings-regular.woff') format('woff'), url('bootstrap/glyphicons-halflings-regular.ttf') format('truetype'), url('bootstrap/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
 }
```
Comment (serhiir) [inline vendor/assets/stylesheets/bootstrap.css]: @anatoliikryvishyn What about this?

**Comment by anatoliikryvishyn** [comment]
@serhiir, updated.

**Comment by serhiir** [comment]
@anatoliikryvishyn I'm wondering, whether `vendor/assets/stylesheets/bootstrap.min.css` file should have any changes after you reverted modifications related to bootstrap?

**Comment by serhiir** [comment]
@anatoliikryvishyn Have you deployed given pull request to the server?

**Comment by anatoliikryvishyn** [comment]
@serhiir, the different in `vendor/assets/stylesheets/bootstrap.min.css` is just a code  formatting.

**Comment by anatoliikryvishyn** [comment]
> @anatoliikryvishyn Have you deployed given pull request to the server?

The [trouble version](https://trouble-tada-interlink.herokuapp.com/) has changes of this pull request. And as this and  [10 pull request](https://github.com/anatoliikryvishyn/TaDa/pull/10) don't intersect. So it`s possible to have 2 pull requests in one [server](https://tada-interlink.herokuapp.com/).

**Comment by serhiir** [comment]
@anatoliikryvishyn Not sure what you meant. Anyway my question is whether you re-deployed changes after the last update of the PR?

**Comment by anatoliikryvishyn** [comment]
@serhiir , let`s end with this PR and then go to 10th. Here is just updated [main server](https://tada-interlink.herokuapp.com/) and here is [empty server](https://trouble-tada-interlink.herokuapp.com/) to see and test changes. Please test.

**Pull request merged**